src/apps/expenses/components/ExpensesSection.js

```diff
@@ -77,48 +92,56 @@ class ExpensesSection extends React.Component {
             `}</style>
             <div className="columns">
               { this.totalExpenses > 0 &&
-                <div id="expenses" className="col">
-                  <h2>
-                    <FormattedMessage
-                      id="collective.expenses.title"
-                      values={{ n: this.totalExpenses }}
-                      defaultMessage={`{n, plural, one {Latest expense} other {Latest expenses}}`}
-                      />
-                  </h2>
+                <div id="expenses" className="first col">
+                  <div className="header">
+                    <h2>
+                      <FormattedMessage
+                        id="collective.expenses.title"
+                        values={{ n: this.totalExpenses }}
+                        defaultMessage="{n, plural, one {Latest expense} other {Latest expenses}}"
+                        />
+                    </h2>
+                    <SmallButton
```

Comment:
Could this be a styled anchor tag somehow? I like having real links for browser accessibility, i.e. opening in a new tab.